<launch>

    <node name="multiclass_segmentation" pkg="igvc_perception" type="multiclass_segmentation.py" output="screen">

        <rosparam param="camera_names">["/cam/center", "/cam/left", "/cam/right"]</rosparam>

        <param name="model_path" value="$(find igvc_sandbox)/models/MC_model.pth" />

        <!-- Use cpu instead of GPU (will run very slowly). -->
        <param name="force_cpu" value="false" />

        <param name="segmentation_topic" value="/segmented/image" />

        <!--Encoder Backbone-->
        <param name="encoder" value="efficientnet-b3"/>
        <param name="encoder_weights" value="imagenet"/>
    </node>

</launch>